livy/src/test/java/org/apache/zeppelin/livy/LivyInterpreterIT.java

```diff
@@ -554,6 +555,17 @@ public void testPySparkInterpreter() throws LivyException {
       assertTrue(result.message().get(0).getData().contains("Traceback"));
     }
 
+    // test utf-8 Encoding
+    try {
+      byte[] utf8Byte = new byte[] {-28,-72,-83,-27,-101,-67};
+      String utf8Str = new String(utf8Byte, StandardCharsets.UTF_8);
```

Comment:
I thought of using Chinese at first but is it OK to put Chinese directly inside source code? Would there be any encoding issue when display or transfer the source code?